apis/apiextensions/v1alpha1/revision_types.go

```diff
@@ -256,6 +256,8 @@ type Patch struct {
 // A FromFieldPathPolicy determines how to patch from a field path.
 type FromFieldPathPolicy string
 
+type MergeOptionsPolicy xpv1.MergeOptions
```

Comment:
I'd prefer to define our own types here. You can see some discussion around this topic at https://github.com/crossplane/crossplane/pull/3315.

Reply:
> But im not sure what you mean directly.

Sorry, disregard my comment - I had misunderstood what you were doing.


> Are there any plans to merge https://github.com/crossplane/crossplane/pull/3315 in time or will this be more a mid-term target?

I'm not completely on-board with the approach taken by #3315 of importing v1 types into a v1alpha package. I believe the types should be decoupled across versions, even if it results in a bunch of duplication and copying logic.